samples/ingestion/ingestion-client/FetchTranscription/Language/AnalyzeConversationsProvider.cs

```diff
@@ -0,0 +1,255 @@
+﻿// <copyright file="AnalyzeConversationsProvider.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace FetchTranscription.Language
```

Comment:
not sure if the distinction between Language and TextAnalytics makes sense. 
I think the logic in here is very similar to the one in TextAnalyticsProvider, is it possible to merge them?

Reply:
I actually had the ConversationAnalysisProvider part of the provider class but moved it out for a couple of reasons:
1. They are separate APIs
2. TA is specifically for text/document capabilities and the new conversations suites are for supporting conversations. The distinction is important as they support different input modalities. In the future I think for speech specifically, all capabilities will need to migrate to the conversation capabilities rather than the TA capabilities as and when we launch more features. 

That said, if it still makes more sense to have them together for a better customer experience, I am happy to merge them.

Reply:
Keeping it separate. Going forward the TA provider will be deprecated in favor of the ConversationAnalysisProvider